RGB 2.2 - RGB traces with solder bridges, not 2.54 headers

climent opened the pull request:
As described, this PR contains the updated RGB with:

Corrected width on GND/VCC tracks
Updated silkscreen with GND marked for LED in both sides
Removed Jumpers and updated with bypass pads.

I do not expect this to be merged, i just created this PR for visibility and to demonstrate the changes I was playing with.

climent committed:
Updated the RGB PCB

Changes:
  * Updated jumpers with bypass tabs. Solder once for bypassing the section.
  * Updated footprints with silkscreen for the GND tab on the LEDs.
  * Cleaned a bunch of tracks, consolidated GND/VCC with consistent width.

climent committed:
More fixes on GND/VCC tracks.

k-loh commented:
I'm looking to place an order of PCBs and am not sure of the current state of SK6812 mini-e support for the Sofle-RGB 2.1

Does this PR contain the all of the necessary PCB changes to have a functional, working board now? Or should I go with the PCB in PR#90 and the code changes mentioned by @Cirromulus?

Cringely commented:
@k-loh, this may no longer be relevant for you but this PR is a cleanup of the current v2.1 PCB and seems like it would still use the existing build guide. #90 is a proposed v3 to update LED support for Mini-E LEDs.

tiadobatima commented:
Howdy @climent... I implemented your idea on top of the `mini-e` LED design (in #90 ).
For now, it's just in this branch: https://github.com/tiadobatima/SofleKeyboard/tree/mini-e%2Bbypass
It makes the whole thing so much simpler. Could you do a sanity check on it, before I open a new PR please?

Thanks!

climent commented:
Yes, I checked and it seems to be correctly implemented. The idea is simply
short the circuit if no LEDs have been added to the "detour". If LED are
added the circuit is complete without the need of closing the jumper.

Thanks for looking into this.

On Sun, 18 Feb 2024 at 03:25, tiadobatima ***@***.***> wrote:

> Howdy @climent <https://github.com/climent>... I implemented your idea on
> top of the mini-e LED design (in #90
> <https://github.com/josefadamcik/SofleKeyboard/pull/90> ).
> For now, it's just in this branch:
> https://github.com/tiadobatima/SofleKeyboard/tree/mini-e%2Bbypass
> It makes the whole thing so much simpler. Could you do a sanity check on
> it, before I open a new PR please?
>
> Thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/josefadamcik/SofleKeyboard/pull/130#issuecomment-1950891879>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFHC6GLF3RLWRM3FMD6WZLYUFYA7AVCNFSM5K23DAR2U5DIOJSWCZC7NNSXTN2JONZXKZKDN5WW2ZLOOQ5TCOJVGA4DSMJYG44Q>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
climent () gmail ! com

DaneEvans commented:
@climent Can you make this an RGB 2.1 folder, 
And assuming you didn't make changes to the plates / svgs etc, drop all of them 
likewise the duplicated library can probably go

I need to fit both this and the v3 one into the directory somehow, so it'll need to be cleaned up

climent commented:
I can work on this in the coming days. I am trying to clean up a bunch of my PRs for the different projects I have been contributing.
